lib/util/authorizer.js

```diff
@@ -142,16 +174,24 @@ module.exports = class Authorizer {
 		return this._keyResolver.getKey(keyId)
 	}
 
+	/**
+	 * @param {any} req Incoming request
+	 */
 	async isAuthorized(req) {
-		const keyResolver = this._keyResolver
-		const parsed = httpSignature.parseRequest(req)
-		const publicKey = await keyResolver.getKey(parsed.keyId)
+		try {
+			const keyResolver = this._keyResolver
+			const parsed = httpSignature.parseRequest(req, undefined)
+			const publicKey = await keyResolver.getKey(parsed.keyId)
 
-		if (httpSignature.verifySignature(parsed, publicKey)) {
-			return true
-		}
+			if (httpSignature.verifySignature(parsed, publicKey)) {
+				return true
+			}
 
-		console.error('Forbidden - failed verifySignature')
-		return false
+			this._logger.error('Forbidden - failed verifySignature')
+			return false
+		} catch (error) {
```

Comment:
`await` silences errors unless explicitly _caught_. This should make it more obvious to sdk users when something went awry (like it did for me when I had pointed to the wrong environment and my `keyId` didn't exist).